templates/zerver/emails/email.css

```diff
@@ -364,57 +246,4 @@ a.button:hover {
         border-radius: 0 !important;
         border-right-width: 0 !important;
     }
-
-    table[class="body"] .btn table {
-        width: 100% !important;
-    }
-
-    table[class="body"] .btn a {
-        width: 100% !important;
-    }
-
-    table[class="body"] .img-responsive {
-        height: auto !important;
-        max-width: 100% !important;
-        width: auto !important;
-    }
-}
-
-@media all {
-    .ExternalClass {
-        width: 100%;
-    }
-
-    .ExternalClass,
-    .ExternalClass p,
-    .ExternalClass span,
-    .ExternalClass font,
-    .ExternalClass td,
-    .ExternalClass div {
-        line-height: 100%;
-    }
-
-    /* iOS converts addresses in emails to links automatically */
-    .apple-link a {
-        color: inherit !important;
-        font-family: inherit !important;
-        font-size: inherit !important;
-        font-weight: inherit !important;
-        line-height: inherit !important;
-        text-decoration: none !important;
-    }
-}
-
-.float-right-image {
-    float: right;
-}
-
-/* The 450px cutoff was determined by testing in Gmail as a point
-   where an image looks good in either state. Logically it makes senes
-   since then the text still has the majority of the space in the
-   side-by-side view. */
-@media only screen and (min-width: 450px) {
-    .float-right-image {
-        max-width: 200px;
-    }
```

Comment:
Let's leave `float-right-image`. We added it in dff5e8ff6e2c34fdcb9738050744d3e2d695a55a -- the purpose is images in hand-written email newsletters sent via `manage.py send_custom_email`. But let's add a comment that this is the use case for it :).